testkit/src/main/kotlin/com/autonomousapps/kit/BuildScript.kt

```diff
@@ -49,12 +49,12 @@ class BuildScript(
 
     val add =
       if (additions.isNotEmpty()) {
-        "\n$additions"
+        "\n$additions\n"
       } else {
         ""
       }
 
-    return buildscriptBlock + pluginsBlock + reposBlock + androidBlock + sourceSetsBlock + featureVariantsBlock + dependenciesBlock + add
+    return buildscriptBlock + pluginsBlock + add + reposBlock + androidBlock + sourceSetsBlock + featureVariantsBlock + dependenciesBlock
```

Comment:
sometimes build script additions should come before other stuff. This might break some tests, won't be sure till I run them. Might need to make this more configurable.

Reply:
I think what I actually need to do is at least a combination of the following:
1. Expose the ability to just write out raw text as a build script for full and transparent configurability
2. Enhance the existing model rather than shoving a bunch of new things into `BuildScript.additions` as raw text.